### PR TITLE
Cleanup localizers

### PR DESCRIPTION
**Pull request by hishamco**
Based on @jtkech comment https://github.com/OrchardCMS/OrchardCore/pull/4708#discussion_r343404259

### CODE REVIEW

**Comment by hishamco** [comment]
Seems there 're many places where localizers are not used, I will remove them after build pass

**Comment by hishamco** [comment]
@jtkech shall I close this, or continue to fix the build and remove unnecessary localizers?

**Comment by jtkech** [comment]
No, i like the `private readonly` field.

In #4708 i also remove some setters for the same reason for properties as `New`, `Logger`, so here you can complete your cleanup fro this kind of properties.

Right now i let in my PR what i already did. Here you change many files that are also updated in my PR, so anyway i think it is better to wait for my PR to be merged and then rebase this one.

**Comment by hishamco** [comment]
> No, i like the private readonly field.

That's what I did

> i think it is better to wait for my PR to be merged and then rebase this one.

I think the other way merging this first will simplify the process, but if you don't like I will wait for yours

**Comment by jtkech** [comment]
Okay, no problem, will see what happens

**Comment by hishamco** [comment]
@jtkech it's ready to merge, then your could rebase on dev

**Comment by jtkech** [comment]
Cool :))

**Comment by hishamco** [comment]
@jtkech i think it's the time to merge this after merging #4708

/cc @sebastienros

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs [hunk 1]
```diff
@@ -23,32 +23,26 @@ public class NodeController : Controller, IUpdateModel
         private readonly IEnumerable<IAdminNodeProviderFactory> _factories;
         private readonly IAdminMenuService _adminMenuService;
         private readonly INotifier _notifier;
-
+        private readonly IHtmlLocalizer H;
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs]: Add a line before the ctor

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs [hunk 2]
```diff
@@ -23,32 +23,26 @@ public class NodeController : Controller, IUpdateModel
...
             _adminMenuService = adminMenuService;
             _authorizationService = authorizationService;
-
             New = shapeFactory;
             _notifier = notifier;
-            T = stringLocalizer;
             H = htmlLocalizer;
         }
 
         public dynamic New { get; set; }
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs]: We can do the same for this property

**Comment by jtkech** [comment]
@hishamco okay, just did a little review, i will finish tomorrow.

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs (hunk 2 above)
Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs]: Ok, I saw you did it as getter property, no worry I will handle them too

**Comment by agriffard** [comment]
Can you please fix the conflicts?

**Comment by hishamco** [comment]
Sure

**Comment by hishamco** [comment]
@agriffard your final review before merge (after build is pass)

**Comment by hishamco** [comment]
Can we merge this?

**Comment by hishamco** [comment]
@jtkech this takes so long, can we merge it?

**Comment by hishamco** [comment]
If it's yes please, let me know. I will send a PR for PoExtractor

**Pull request merged**